dist/css/bootstrap.css

```diff
@@ -6507,6 +6541,19 @@ tbody.collapse.in {
   right: auto;
   left: 0;
 }
+@media (max-width: 576px) {
+  .dropdown-menu-native {
+    position: fixed;
+    top: auto;
+    bottom: 0;
+    width: 100%;
+    max-width: 100% !important;
+    min-height: 200px;
+    max-height: 50% !important;
+    -webkit-box-shadow: 0 20px 66px 0 #223049;
+            box-shadow: 0 20px 66px 0 #223049;
```

Comment:
Is this the same box shadow or a new one?  Not sure how relevant it is in this view?

Reply:
It's the same colour but with 100% opacity. I asked a designer and he told me to make the shadow stronger on top so the menu doesn't blend in too much with a white background